app.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 
   // List of chainIds which metadata cache queries will return by default.
   // This preselects the Chains user preferences.
-  chainIds: [1, 137, 56, 246, 1285],
+  chainIds: [1, 137, 56, 246, 1285, 8996],
```

Comment:
What network is it that you're adding here? It's not in the list of supported chainIds so I'm not sure why you are adding it here in the pre-selected list?

Reply:
that is barge, definitely can be removed